### DISCUSSION TITLE
How can I get symbol shape of a layer as input of another layer in run time? Not with ”infer_shape“

### DISCUSSION THREAD

**Discussion by summerlvsong** [apache/incubator-mxnet#8448] [Q&A]
## Description
When I try to use  "mx.symbol.Deconvolution", I find it hard to set the the parameter "target_shape". I want it to deconvolute the input to a specific shape of a former layer, So I think I have to set "target_shape" to the shape of the former layer, It is right?
However, I cannot find a way to acquire the output shape of the former layer in run time.

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.
